### PR TITLE
*: add counter benchmark.

### PR DESCRIPTION
**Pull request by siddontang**
@overvenus

### CODE REVIEW

**Comment by siddontang** [comment]
Hi @lucab 
Is is ok to add `test` crate directly here?

File: benches/metrics/mod.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod counter;
```
Comment (siddontang) [inline benches/metrics/mod.rs]: seems that we can move counter to parent benches directly?

**Comment by disksing** [comment]
LGTM

**Comment by overvenus** [comment]
LGTM

**Pull request merged**